content/en/blog/2022/prow-and-tide-for-kubernetes-contributors/prow-and-tide-for-kubernetes-contributors.md

```diff
@@ -0,0 +1,48 @@
+
+That's right. Squashing your commits is a label away and the tooling will do the rest. To do this on your PR you'll need to comment with the following:
+
+`/label tide/merge-method-squash`
+
+These two pages document all the functionality available to Kubernetes contributors (either through labels or Prow). The biggest thing to remember is when a something is a label or a command:
+
+* [Prow Command Help](https://prow.kubernetes.io/command-help)
+* [test-infra/label_sync/labels.md](https://github.com/kubernetes/test-infra/tree/master/label_sync) (remember, all labels are preceded with `/label`)
+
+Some of my more often used commands and labels:
+
+* `/assign` (using it without adding a name assigns yourself)
+* `/honk`
+* `/(woof|bark|this-is-{fine|not-fine|unbearable})`
+* `/remove-lifecycle stale` (when issues aren't touched for a period of time they're marked stale)
+* `/shrug`
+* `/label area/contributor-comms` (You can use this to flag down the contributor communications team for reviews, comments on any issue, feedback, etc.)
+* `/label size/X` (Sizes are assigned automatically based on the number of lines changed in the PR)
+* `/label do-not-merge/hold` (This one is used for many things; if your PR is a work in progress, needs to be held to a certain date, etc.)
+* `/lgtm` (Adds or removes the 'lgtm' label which is typically used to gate merging)
+* `/approve` (Approves a pull request; must be done by someone in the repo's OWNERS file)
+
+What if you need a label that isn't available on a certain GitHub repository? I'm glad you asked! This PR demonstrates how to add labels to a repo: [https://github.com/kubernetes/test-infra/pull/24315](https://github.com/kubernetes/test-infra/pull/24315). You'll need to update the [labels.yaml file](https://github.com/kubernetes/test-infra/blob/master/label_sync/labels.yaml) (the configuration) and the [labels.md file](https://github.com/kubernetes/test-infra/blob/master/label_sync/labels.md) (documentation).
+
+I've done this once in five years of contributing. But, it's good to write it down as it's something that isn't as trivial as you think because of the importance of the label_sync tooling.
+
+These are a handful of the [commands](https://prow.kubernetes.io/command-help) and [labels](https://github.com/kubernetes/test-infra/blob/master/label_sync/labels.md) I enjoy. I'm sure there are many others that are helpful to folks. With that in mind, see if there's something you can benefit from in these resources. They are there to make working on Kubernetes a better experience. If you think there's some functionality missing, I'd invite you to drop a Slack message in [SIG ContribEx](https://kubernetes.slack.com/archives/C1TU9EB9S) or [SIG K8s Infra](https://kubernetes.slack.com/archives/CCK68P2Q2) to discuss.
```

Comment:
SIG Testing is the other group that handles labels. SIG K8s Infra only operates the underlying infra.